src/dmd/utils.d

```diff
@@ -132,3 +132,9 @@ extern (C++) void escapePath(OutBuffer* buf, const(char)* fname)
         fname++;
     }
 }
+
+/// Slices a `\0`-terminated C-string, excluding the terminator
+const(char)[] toDString (const(char)* s) pure nothrow @nogc
```

Comment:
Why not `toString`?

Reply:
`toString` is to provide a string representation, this is to convert a `const(char)*` to a `const(char)[]`.